deployment/docker-compose/README.md

```diff
@@ -0,0 +1,458 @@
+- Provider: 
+    - ALERT_PROVIDER_BLACKDUCK_BLACKDUCK_URL= The URL for the Black Duck server.
+- Channel:
+    - ALERT_CHANNEL_JIRA_CLOUD_JIRA_CLOUD_URL= The URL for the Jira Cloud server.
+    - ALERT_CHANNEL_EMAIL_MAIL_SMTP_HOST= The SMTP host used to send email messages.
+- Component: 
+    - ALERT_COMPONENT_SETTINGS_SETTINGS_LDAP_ENABLED= Boolean to determine if LDAP authentication is used.
+
+## Advanced Configuration
+This section describes some advanced configuration settings for the Alert server.
+
+### Changing Server Port
+If Alert should not be running on it's default port of 8443, then this section describes what you have to change in order to use a different port.
+
+For this advanced setting since there are more than just environment variables that need to be set this should be performed by editing the ```docker-compose.local-overrides.yml``` file.
+
+- Overrides File Changes.
+    - Define the new ports for the alert service.  Add 'ports' to the service description. 
+    ```
+        alert: 
+            ports: ['<NEW_PORT>:<NEW_PORT>']
+    ```
+    - Define the ```ALERT_SERVER_PORT``` environment variable.
+    ```
+        alert: 
+            environment:
+                - ALERT_HOSTNAME=localhost
+                - ALERT_SERVER_PORT=<NEW_PORT>
+    ```
+    - Define the healthcheck for the alert service. Add 'healthcheck' to the service description.
+    ```
+        alert:
+            healthcheck:
+                  test: [CMD, /usr/local/bin/docker-healthcheck.sh, 'https://localhost:<NEW_PORT>/alert/api/about',
+                         /opt/blackduck/alert/security/root.crt, /opt/blackduck/alert/security/blackduck_system.crt,
+                         /opt/blackduck/alert/security/blackduck_system.key]
+                  interval: 30s
+                  timeout: 60s
+                  retries: 15
+    ```
+Example:
+- Change the port to 9090 via the ```docker-compose.local-overrides.yml``` file.
+```
+    alert:
+        ports: ['9090:9090']
+        environment:
+            - ALERT_HOSTNAME=localhost
+            - ALERT_SERVER_PORT=9090
+        healthcheck:
+            test: [CMD, /usr/local/bin/docker-healthcheck.sh, 'https://localhost:9090/alert/api/about',
+                 /opt/blackduck/alert/security/root.crt, /opt/blackduck/alert/security/blackduck_system.crt,
+                /opt/blackduck/alert/security/blackduck_system.key]
+            interval: 30s
+            timeout: 60s
+            retries: 15
+```
+  
+Note: Work with your IT staff if necessary to verify the configured port is accessible through the network.
+
+### Changing Memory Settings
+If Alert should be using more memory than its default settings, then this section describes what you have to change in order to allocate more memory.
+
+For this advanced setting since there are more than just environment variables that need to be set this should be performed by editing the ```docker-compose.local-overrides.yml``` file.
```

Comment:
This can probably be clarified with commas "For this advanced setting, since there are more than just environment variables that need to be set, edit the ```docker-compose.local-overrides.yml``` file."